config/routes.rb

```diff
@@ -1,7 +1,11 @@
 Rails.application.routes.draw do
+  root "enapsa_monthly_reports#new"
+  get "/enapsa_monthly_report/new", to: "enapsa_monthly_reports#new"
+  post "/enapsa_monthly_report/create", to: "enapsa_monthly_reports#create"
+  get "/enapsa_monthly_report/show", to: "enapsa_monthly_reports#show"
   resources :employers
+  get "/enapsa_monthly_report/download_csv", to: "enapsa_monthly_reports#download_csv" #  needs refactoring
```

Comment:
How do you feel about using [resourceful routing](https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default)?